### ISSUE TITLE
[Tree] aria-level should be depth + 1 (according to spec values should be >= 1).

### ISSUE THREAD

**Issue by nchevobbe** [devtools-html/devtools-core#963]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1438837
